client/src/components/Login.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import Modal from 'plaid-threads/Modal';
+import ModalBody from 'plaid-threads/ModalBody';
+import Button from 'plaid-threads/Button';
+import TextInput from 'plaid-threads/TextInput';
+
+import { useCurrentUser } from '../services';
+
+const Login = () => {
+  const { login } = useCurrentUser();
+  const [show, setShow] = useState(false);
+  const [value, setValue] = useState('');
+
+  const handleSubmit = () => {
+    setShow(false);
+    login(value);
+  };
+
+  return (
+    <div>
+      <Button centered inline onClick={() => setShow(!show)}>
+        Login
+      </Button>
+      <Modal isOpen={show} onRequestClose={() => setShow(false)}>
+        <>
+          <ModalBody
+            onClickCancel={() => setShow(false)}
+            header="User Login"
+            isLoading={false}
+            onClickConfirm={handleSubmit}
+            confirmText="Submit"
+            body={
+              <TextInput
+                id="id-6"
+                placeholder="Enter User Name"
+                value={value}
+                onChange={e => setValue(e.currentTarget.value)}
+              />
+            }
+          />
+        </>
```

Comment:
a fragment is useful for providing a single jsx element to wrap two children (for example in the render method of a component), but its not needed here. here it is just surrounding a single element so its not doing anything, and Modal can have multiple children anyways